src/utils/generateHTML.js

```diff
@@ -0,0 +1,138 @@
+const Manager = require("../lib/Manager");
+const Engineer = require("../lib/Engineer");
+const Intern = require("../lib/Intern");
+const Employee = require("../lib/Employee");
+
+//make manager card
+const generateManagerCard = (manager) => {
```

Comment:
You have duplication here and in your `cards` folder. I think it's the other one which is being used, so please remove whichever isn't being used